src/Tribe/Service_Providers/Context.php

```diff
@@ -270,6 +270,38 @@ static function ( $post_type ) {
 					Tribe__Context::REQUEST_VAR => [ TEC::TAXONOMY ],
 				],
 			],
+			'view_url' => [
```

Comment:
:warning: Array double arrow not aligned correctly; expected 13 space(s) between "'view_url'" and double arrow, but found 1.

<sub>hash: 99520f304f2e851d2776ff4d484a1db9</sub>